src/main/java/com/example/Twitter/controller/TwitterController.java

```diff
@@ -0,0 +1,41 @@
+package com.example.Twitter.controller;
+import org.springframework.web.bind.annotation.PathVariable;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RequestMethod;
+import org.springframework.web.bind.annotation.RestController;
+import twitter4j.Status;
+import twitter4j.Twitter;
+import twitter4j.TwitterException;
+import twitter4j.TwitterFactory;
+import java.util.List;
+
+@RestController
+@RequestMapping("/twitter")
+public class TwitterController {
+
+    @RequestMapping(value = "/getTimeLine", method = RequestMethod.GET)
+    public List<Status> getFeed() throws TwitterException
+    {
+            Twitter twitter = TwitterFactory.getSingleton();
```

Comment:
Is it possible to create a single twitter instance outside of both the methods and reuse the same for both?

Reply:
Yes, It is posssible. I will create single instance and commit the changes.